src/home_robot_hw/README.md

```diff
@@ -45,7 +62,11 @@
 
 ## Usage
 
-### Launching the Hello Stretch hardware drivers
+### On robot: Launching the Hello Stretch hardware drivers
 ```sh
 roslaunch home_robot startup_stretch_hector_slam.launch
-```
\ No newline at end of file
+```
+
+### On user machine: Interacting with the robot via the client
```

Comment:
standardizing terms - "user machine" vs "server" or "workstation"?

Reply:
I don't like the term "server" for the user machine as the robot is more of a server on the control side as it runs rosmaster (also I like the robotics as a service (RAAS) design methodology).
I'm using "robot" vs "user machine" for now but "workstation" also seems fine, not sure how ambiguous it is.
Would be nice to also get @zephirefaith 's input on this.